subsystems/motor/speedmodifiers/CapSpeedModifier.java

```diff
@@ -0,0 +1,21 @@
+package org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers;
+
+
+import org.usfirst.frc4904.standard.Util;
+
+public class CapSpeedModifier implements SpeedModifier {
+	public Util.Range speedCap;
+	
+	public CapSpeedModifier(double minSpeed, double maxSpeed) {
+		speedCap = new Util.Range(minSpeed, maxSpeed);
```

Comment:
Perhaps use a self constructor (e.g. replace with `this(new Util.Range(minSpeed, maxSpeed));`)